phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/ViewEngines.java

```diff
@@ -159,8 +155,14 @@ private void showEngineDetails(EngineDTO engineDTO) {
         showRightView(currentEnginePanel);
     }
 
-    public void setOnSelectCategory(Consumer<EngineCategoryDTO> onSelectCategory) {
-        this.onSelectCategory = onSelectCategory;
+    private void processFilterText(String filterText) {
```

Comment:
I don't know if we want to add the `pause` to the `SearchBox`. Currently we're only using `pause` this, because the lists we're filtering with the search term are quite long which makes it work intensive.